erpnext/accounts/doctype/payment_entry/payment_entry_list.js

```diff
@@ -1,12 +1,52 @@
 frappe.listview_settings['Payment Entry'] = {
 
-	onload: function(listview) {
-		listview.page.fields_dict.party_type.get_query = function() {
+	onload: function (listview) {
+		listview.page.fields_dict.party_type.get_query = function () {
 			return {
 				"filters": {
 					"name": ["in", Object.keys(frappe.boot.party_account_types)],
 				}
 			};
 		};
+		listview.page.add_actions_menu_item(__('Print Check'), () => {
```

Comment:
We're not actually printing anything, we should call this something else

```suggestion
		listview.page.add_actions_menu_item(__('Set Cheque Numbers'), () => {
```